Source/Core/Core/AchievementManager.cpp

```diff
@@ -1337,6 +1338,11 @@ void AchievementManager::ActivateDeactivateAchievement(AchievementId id, bool en
   {
     rc_runtime_activate_achievement(&m_runtime, id, m_game_data.achievements[index].definition,
                                     nullptr, 0);
+    if (rc_runtime_get_achievement_measured(&m_runtime, id, &status.measured_value,
+                                            &status.measured_target) == 0)
+    {
+      WARN_LOG_FMT(ACHIEVEMENTS, "Failed to get measured data for achievement ID {}.", id);
```

Comment:
Should the failure case reset `measured_value` and `measured_target` to a sensible default? Or maybe store a flag so that `GetAchievementProgress()` knows when it gets invalid return values?

Reply:
The function sets both to zero unless the specific error is that one of them is a nullptr, and target=0 is what non-measured achievements always return anyways, so a failure here would just mean that the achievement doesn't show a progress bar at all.